Use consistent spacing and indentation (Flake8 errors)

After running Flake8 (#33), we received several notices about **spaces and indentation** in CiviWiki sourcecode.

# Goal
Use consistent indentation and spacing throughout CiviWiki sourcecode. Conform to guidelines of PEP-8.

# Task
Clean up the following Flake8 errors:

- [x] ./api/read.py:220:60: E272 multiple spaces before keyword                                                                                                                                                                                                                      
- [x] ./api/read.py:220:87: E272 multiple spaces before keyword
- [x] ./api/write.py:499:19: E271 multiple spaces after keyword
- [x] ./api/write.py:502:58: E225 missing whitespace around operator
- [x] ./api/write.py:549:19: E271 multiple spaces after keyword
- [x] ./api/models/civi.py:41:1: E101 indentation contains mixed spaces and tabs
- [x] ./api/models/civi.py:41:1: W191 indentation contains tabs
- [x] ./api/models/civi.py:43:1: E101 indentation contains mixed spaces and tabs
- [x] ./api/models/civi.py:54:31: E251 unexpected spaces around keyword / parameter equals
- [x] ./api/models/civi.py:65:1: E101 indentation contains mixed spaces and tabs
- [x] ./api/models/civi.py:65:1: W191 indentation contains tabs
- [x] ./api/models/civi.py:67:1: E101 indentation contains mixed spaces and tabs
- [x] ./api/models/civi.py:116:29: E241 multiple spaces after ':'
- [x] ./api/models/civi.py:124:5: E301 expected 1 blank line, found 0
- [x] ./api/models/civi.py:156:39: E225 missing whitespace around operator
- [x] ./api/models/civi.py:156:42: E221 multiple spaces before operator
- [x] ./api/models/civi.py:166:48: E225 missing whitespace around operator
- [x] ./api/models/civi.py:190:1: E101 indentation contains mixed spaces and tabs
- [x] ./api/models/civi.py:190:1: W191 indentation contains tabs
- [x] ./api/models/civi.py:191:1: E101 indentation contains mixed spaces and tabs
- [x] ./api/models/civi.py:208:1: E305 expected 2 blank lines after class or function definition, found 1
- [x] ./api/models/account.py:88:1: E305 expected 2 blank lines after class or function definition, found 1
- [x] ./api/models/account.py:169:5: E301 expected 1 blank line, found 0
- [x] ./api/models/thread.py:57:1: E305 expected 2 blank lines after class or function definition, found 1
- [x] ./authentication/forms.py:121:297: E251 unexpected spaces around keyword / parameter equals
- [x] ./authentication/forms.py:121:299: E251 unexpected spaces around keyword / parameter equals
- [x] ./authentication/forms.py:121:316: E251 unexpected spaces around keyword / parameter equals
- [x] ./authentication/forms.py:121:318: E251 unexpected spaces around keyword / parameter equals
- [x] ./authentication/authentication.py:29:5: E301 expected 1 blank line, found 0
- [x] ./authentication/authentication.py:39:1: E305 expected 2 blank lines after class or function definition, found 1

--- a/project/api/models/account.py
+++ b/project/api/models/account.py
@@ -85,6 +85,7 @@ class PathAndRename(object):
         filename = '{}.{}'.format(new_filename, ext)
         return os.path.join(self.sub_path, filename)
 
+
 profile_upload_path = PathAndRename('')
 
 
@@ -166,6 +167,7 @@ class Account(models.Model):
         super(Account, self).save(*args, **kwargs)
     #
     #
+    
     def resize_profile_image(self):
         profile_image_field = self.profile_image
         image_file = StringIO.StringIO(profile_image_field.read())

--- a/project/api/models/civi.py
+++ b/project/api/models/civi.py
@@ -38,7 +38,7 @@ class CiviManager(models.Manager):
             "votes": civi.votes(),
             "id": civi.id,
             "thread_id": civi.thread.id
-	    }
+        }
 
         if filter and filter in data:
             return json.dumps({filter: data[filter]})
@@ -51,7 +51,7 @@ class CiviManager(models.Manager):
             "body": civi.body,
             "author": dict(
                 username=civi.author.user.username,
-                profile_image= civi.author.profile_image.url if civi.author.profile_image else "/media/profile/default.png",
+                profile_image=civi.author.profile_image.url if civi.author.profile_image else "/media/profile/default.png",
                 first_name=civi.author.first_name,
                 last_name=civi.author.last_name
             ),
@@ -62,7 +62,7 @@ class CiviManager(models.Manager):
             "id": civi.id,
             "thread_id": civi.thread.id,
             "links": [c for c in civi.linked_civis.all().values_list('id', flat=True)]
-	    }
+        }
 
         if filter and filter in data:
             return json.dumps({filter: data[filter]})
@@ -113,7 +113,7 @@ class Civi(models.Model):
             'total': act_votes.count() - act_votes.filter(activity_type='vote_neutral').count(),
             'votes_vneg': act_votes.filter(activity_type='vote_vneg').count(),
             'votes_neg': act_votes.filter(activity_type='vote_neg').count(),
-            'votes_neutral':  act_votes.filter(activity_type='vote_neutral').count(),
+            'votes_neutral': act_votes.filter(activity_type='vote_neutral').count(),
             'votes_pos': act_votes.filter(activity_type='vote_pos').count(),
             'votes_vpos': act_votes.filter(activity_type='vote_vpos').count()
         }
@@ -121,6 +121,7 @@ class Civi(models.Model):
 
     created = models.DateTimeField(auto_now_add=True, blank=True, null=True)
     last_modified = models.DateTimeField(auto_now=True, blank=True, null=True)
+
     def _get_created_date_str(self):
         d = self.created
         return "{0} {1}, {2}".format(month_name[d.month], d.day, d.year)
@@ -153,7 +154,7 @@ class Civi(models.Model):
         if x > 0:
             v = votes['total'] if votes['total'] > 1 else 2
             #step3 - A X*Log10V+Y + F + (##/T) = Rank Value
-            rank = x * math.log10(v) *amp  + y + f + g/t
+            rank = x * math.log10(v) * amp + y + f + g/t
 
         elif x == 0:
 
@@ -163,7 +164,7 @@ class Civi(models.Model):
             v = votes['total'] if votes['total'] > 1 else 2
             #step3 - C
             if abs(x)/v <= 5:
-                rank = abs(x) * math.log10(v) *amp + y + f + g/t
+                rank = abs(x) * math.log10(v) * amp + y + f + g/t
             else:
                 rank = x * math.log10(v) * amp + y + f + g/t
         return rank
@@ -187,7 +188,7 @@ class Civi(models.Model):
             # Not Implemented Yet
             "hashtags": [],
             "attachments": [{'id': img.id, 'url': img.image_url} for img in self.images.all()],
-	    }
+        }
         if req_acct_id:
             data['score'] = self.score(req_acct_id)
 
@@ -204,6 +205,7 @@ class PathAndRename(object):
         new_filename = str(uuid.uuid4())
         filename = '{}.{}'.format(new_filename, ext)
         return os.path.join(self.sub_path, filename)
+
 
 image_upload_path = PathAndRename('')
 

--- a/project/api/models/thread.py
+++ b/project/api/models/thread.py
@@ -54,6 +54,7 @@ class PathAndRename(object):
         filename = '{}.{}'.format(new_filename, ext)
         return os.path.join(self.sub_path, filename)
 
+
 image_upload_path = PathAndRename('')
 
 class Thread(models.Model):

--- a/project/api/read.py
+++ b/project/api/read.py
@@ -217,7 +217,7 @@ def get_thread(request, thread_id):
             'hashtags': t.hashtags.all().values(),
             'author': {
                 'username': t.author.user.username,
-                'profile_image': t.author.profile_image.url  if t.author.profile_image  else "/media/profile/default.png",
+                'profile_image': t.author.profile_image.url if t.author.profile_image else "/media/profile/default.png",
                 'first_name': t.author.first_name,
                 'last_name': t.author.last_name
             },

--- a/project/api/write.py
+++ b/project/api/write.py
@@ -496,10 +496,10 @@ def uploadCiviImage(request):
             data = {
                 "attachments": [{'id': img.id, 'url': img.image_url} for img in c.images.all()],
             }
-            return  JsonResponse(data)
+            return JsonResponse(data)
 
         except Exception as e:
-            return HttpResponseServerError(reason=(str(e)+ civi_id + str(request.FILES)))
+            return HttpResponseServerError(reason=(str(e) + civi_id + str(request.FILES)))
     else:
         return HttpResponseForbidden('allowed only via POST')
 
@@ -546,7 +546,7 @@ def uploadThreadImage(request):
             data = {
                 'image': thread.image_url
             }
-            return  JsonResponse(data)
+            return JsonResponse(data)
 
         except Exception as e:
             return HttpResponseServerError(reason=(str(e)))

--- a/project/authentication/authentication.py
+++ b/project/authentication/authentication.py
@@ -26,6 +26,7 @@ from django.template.response import TemplateResponse #TODO: move this out to vi
 class AccountActivationTokenGenerator(PasswordResetTokenGenerator):
     """ Token Generator for Email Confirmation """
     key_salt = "django.contrib.auth.tokens.PasswordResetTokenGenerator"
+
     def _make_token_with_timestamp(self, user, timestamp):
         """ Token function pulled from Django 1.11 """
         ts_b36 = int_to_base36(timestamp)
@@ -35,6 +36,7 @@ class AccountActivationTokenGenerator(PasswordResetTokenGenerator):
             unicode(user.pk) + unicode(timestamp)
         ).hexdigest()[::2]
         return "%s-%s" % (ts_b36, hash)
+
 
 account_activation_token = AccountActivationTokenGenerator()
 

--- a/project/authentication/forms.py
+++ b/project/authentication/forms.py
@@ -118,7 +118,7 @@ class RecoverUserForm(AuthRecoverUserForm):
             domain = get_current_site(request).domain
             base_url = "http://{domain}/auth/password_reset/{uid}/{token}/"
             url_with_code = base_url.format(domain=domain, uid=uid, token=token)
-            email_body = "You're receiving this email because you requested an account recovery email for your user account at {domain}. Your username for this email is: {username}. If you also need to reset your password, please go to the following page and choose a new password.".format(domain = domain, username = user.username)
+            email_body = "You're receiving this email because you requested an account recovery email for your user account at {domain}. Your username for this email is: {username}. If you also need to reset your password, please go to the following page and choose a new password.".format(domain=domain, username=user.username)
 
             email_context = {
                 'title' : 'Account Recovery for CiviWiki',

--- a/project/civiwiki/wsgi.py
+++ b/project/civiwiki/wsgi.py
@@ -6,13 +6,10 @@ It exposes the WSGI callable as a module-level variable named ``application``.
 For more information on this file, see
 https://docs.djangoproject.com/en/1.8/howto/deployment/wsgi/
 """
-
+from django.core.wsgi import get_wsgi_application
+from whitenoise.django import DjangoWhiteNoise
 import os
 
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "civiwiki.settings")
-
-from django.core.wsgi import get_wsgi_application
-from whitenoise.django import DjangoWhiteNoise
-
 application = get_wsgi_application()
 application = DjangoWhiteNoise(application)

